### PR TITLE
feat(app): add `include` to `RedirectOptions`

### LINKED ISSUE #268: Everything to be excluded in redirectOptions except one folder
Hello guys,
I am trying to make sure the authentication check if the user is logged in to be applicable to only one folder and every other to be excluded from it. Should I use some regex if it is possible? Or should I write down every route that should be excluded?

### PR DESCRIPTION
**Pull request by jojomatik**
Closes #268

## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Description

Add an `include` option to `RedirectOptions`. If set, only routes matching the supplied regex are considered for redirects.

## Checklist:

- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes (if not applicable, please state why)

### CODE REVIEW

File: src/runtime/plugins/auth-redirect.ts [hunk 1]
```diff
@@ -8,9 +8,20 @@ export default defineNuxtPlugin({
       'global-auth',
       defineNuxtRouteMiddleware((to) => {
         const config = useRuntimeConfig().public.supabase
-        const { login, callback, exclude, cookieRedirect } = config.redirectOptions
+        const { login, callback, include, exclude, cookieRedirect } = config.redirectOptions
         const { cookieName, cookieOptions } = config
 
+        // Redirect only on included routes (if defined)
+        if (include && include.length > 0) {
+          const isIncluded = [...include].some((path) => {
```
Comment (larbish) [inline src/runtime/plugins/auth-redirect.ts:16]:
```suggestion
          const isIncluded = include.some((path) => {
```

File: playground/pages/protected/index.vue
Comment (larbish) [inline playground/pages/protected/index.vue:1]: Can you create `pages/protected.vue` instead? (just to be consistent with the `unprotected` route)

**Review by larbish — COMMENTED** [review]
Thanks for the PR! LGTM except two little things. If you can just update, I'll merge then

**Comment by jojomatik** [comment]
Thanks for the feedback. I've force pushed with the two changes :)

**Review by larbish — APPROVED** [review]

**Pull request merged**